packages/xsnap/test/xsnap-eof.test.js

```diff
@@ -0,0 +1,181 @@
+import test from 'ava';
+import * as proc from 'child_process';
+import * as os from 'os';
+import fs from 'fs';
+import { fileURLToPath, URL } from 'url';
+import { tmpName } from 'tmp';
+import { text } from 'stream/consumers';
+import { makeNetstringWriter } from '@endo/netstring';
+import { makeNodeWriter } from '@endo/stream-node';
+import { makePromiseKit } from '@endo/promise-kit';
+import { options } from './message-tools.js';
+import { xsnap, QUERY_RESPONSE_BUF } from '../src/xsnap.js';
+
+test('xsnap-worker complains while waiting for answer when parent is killed', async t => {
+  const exitedPKit = makePromiseKit();
+
+  const p =
```

Comment:
This `p` name seems overly generic (and we tend to use "P" as a suffix for Promises and Presences, of which this is neither).
```suggestion
  const child =
```